web/client-api/src/main/java/io/deephaven/web/client/api/subscription/ViewportRow.java

```diff
@@ -65,6 +70,37 @@ public Format getFormat(Column column) {
             JsArray<Any> formatStrings = Js.uncheckedCast(dataColumns[column.getFormatStringColumnIndex()]);
             formatString = formatStrings.getAtAsAny(offsetInSnapshot).asString();
         }
-        return new Format(cellColors, rowColors, numberFormat, formatString);
+        if(column.getFormatDatabarColumnIndexRange() != null) {
+            Integer[] range = column.getFormatDatabarColumnIndexRange();
+            Integer start = range[0];
+            Integer end = range[1];
+
+            JsArray<Any> formatDatabarStrings = Js.uncheckedCast(dataColumns[start]);
+            formatDatabarString = formatDatabarStrings.getAtAsAny(offsetInSnapshot).asString();
```

Comment:
I think this line is causing the error below since disabling that line makes the error go away

<img width="1717" alt="image" src="https://github.com/deephaven/deephaven-core/assets/55671206/b2dcb755-6d87-41ed-9d78-6ad5514b7813">